src/loki/worker.cc

```diff
@@ -357,6 +357,12 @@ loki_worker_t::work(const std::list<zmq::message_t>& job,
         result.messages.emplace_back(request.SerializeAsString());
         break;
       case Options::isochrone:
+// return early if geotiff was requested but GDAL isn't enabled
+#ifndef ENABLE_GDAL
```

Comment:
be prepared to see plenty of these... I'm open to any suggestions that reduce these wherever possible